src/libsync/owncloudtheme.cpp

```diff
@@ -48,4 +46,15 @@ QIcon ownCloudTheme::aboutIcon() const
 {
     return Resources::themeUniversalIcon(QStringLiteral("oc-image-about"));
 }
+
+QmlButtonColor ownCloudTheme::brandedButtonColor() const
+{
+    const QColor button("#709cd2");
+    return {button, Qt::white, button.darker()};
+}
+
+QmlButtonColor ownCloudTheme::brandedButtonColor2() const
```

Comment:
2?

Reply:
Its defining the color for the less important login button, so just a  second pair of colors.
No idea how to call that.

Reply:
Maybe `primaryColor` and `secondaryColor`?